### DISCUSSION TITLE
DISCUSSION: Difficulty Tables

### DISCUSSION THREAD

**Discussion by wcko87** [wcko87/beatoraja-english-guide#26] [Guide discussion]
Thread for discussing this specific wiki page.

**Comment by hadronyche** [comment]
What is the best way to create a local table/folder for beatoraja? With BeMusicSeeker it's as simple as dragging and dropping charts into a playlist. Is there a way to convert BMSeeker tables/json to, or use them within, beatoraja?

**Reply by wcko87** [reply]
beatoraja has native support for tables (see <https://github.com/wcko87/beatoraja-english-guide/wiki/Difficulty-Tables#using-difficulty-tables-in-beatoraja>), so BeMusicSeeker isn't required to use tables in beatoraja.

**Reply by hadronyche** [reply]
Sure, but bemusicseeker can create json tables, I shouldn't need to host a page just to get my own table into beatoraja. Is there a way to do this locally?

Edit: To clarify, I want to create my own table from scratch, not import an existing one from BMSeeker or add the URL in the config.

**Reply by wcko87** [reply]
beatoraja allows you to create your own custom tables via the "table" tab in the configuration window (you can also alt tab to it while the game is running).
I use it to organize my charts like an extended favorites list.
Currently you can't export or import tables from there, but it's something I'm currently looking into improving since I use this feature a lot.

**Reply by hadronyche** [reply]
Ah, I missed that tab. I'll have a look, thanks.

**Reply by wcko87** [reply]
I created an image explaining how to use the custom tables feature some time ago, so I'll post it here for future reference.

![image](https://user-images.githubusercontent.com/27341392/136740855-eeb969b4-a4fa-4254-9fc0-eac673b4d0c0.png)

**Reply by hadronyche** [reply]
Makes sense, I agree it looks like it could do with some improvements e.g. for myself it would be useful to have a chart path column (e.g. if you wanted to find all the charts/songs from a specific event or folder), and filtering by level ranges, etc. It also looks like you can't operate on more than one table at once? I see you can create additional .json in the table folder, and they will load, but only default.json will show in the Table tab?

**Reply by wcko87** [reply]
Yeah, multiple tables is the first thing I'm looking into at the moment.
